libs/core/files-manager/reducers/files-manager.reducer.ts

```diff
@@ -136,7 +136,7 @@ export const filesManagerReducer = createReducer<FilesManagerState>(
         }
       })
       .addCase(changeLocation, (state) => {
-        return { ...state, selectedItems: { rows: [] } }
+        return { ...initialState }
```

Comment:
1) maybe it would be good idea to remove 'changeLocation' and instead call setInitialFilesManagerState?
2) why we no longer need to keep actual state and clear selected items?

Reply:
"changeLocation" is a global action triggered by one of the listeners, and I don't want to modify it, especially not in this pull request. My changes are specific to the current functionality of the application, which I didn't intend to alter in this particular fix.